taxi/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+

```

Comment:
There no need to have this blank line here. Thiso two imports related to the same import group